shell/platform/embedder/embedder.h

```diff
@@ -505,6 +505,9 @@ typedef struct {
   // |FlutterEngineRun| call is made.
   FlutterUpdateSemanticsCustomActionCallback
       update_semantics_custom_action_callback;
+  // Path to a directory used to store data that is cached across runs of a
+  // Flutter application (such as compiled shader programs used by Skia).
+  const char* persistent_cache_path;
```

Comment:
Can you mentioned that this is optional and that the string must be NULL terminated (see documentation for the members above).

Reply:
done